inc/admin/upgrader.php

```diff
@@ -143,6 +143,21 @@ function rocket_first_install() {
 			]
 		)
 	);
+
+	add_option(
+		rocket_get_constant( 'WP_ROCKET_DEBUG_SLUG' ),
+		/**
+		 * Filters the default rocket debug options array.
+		 *
+		 * @param array Array of default rocket debug options.
+		 */
+		apply_filters(
+			'rocket_first_install_debug_options',
+			[
+				'last_rucss_job_added' => '',
+			]
+		)
+	);
```

Comment:
Couple of things here:
- We should do this in a separate method in a debug subscriber
- We need to check that it's only done once when upgrading, by using the plugin version check